### PR TITLE
chore: Update README.md

### PR DESCRIPTION
**Pull request by feng-tao**
### Summary of Changes
fix the format

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [ ] PR title addresses the issue accurately and concisely, including a title prefix.
- [ ] PR includes a summary of changes.
- [ ] My commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ Please visit [Roadmap](https://github.com/lyft/amundsen/blob/master/docs/roadmap
...
 - Towards Enterprise Grade Data Discovery and Data Lineage with Apache Atlas and Amundsen by [Verdan Mahmood](https://github.com/verdan) and Marek Wiewiorka from ING {[slides](https://docs.google.com/presentation/d/1FixTTNd1dt_f3PAKhL1KLOeOLsIQq0iFvQA6qlpjIg0/edit#slide=id.p1), [talk](https://bigdatatechwarsaw.eu/agenda/)} (Big Data Technology Warsaw Summit 2020)
 - Airflow @ Lyft (which covers how we integrate Airflow and Amundsen) by [Tao Feng](https://github.com/feng-tao) {[slides](https://www.slideshare.net/taofung/airflow-at-lyft-airflow-summit2020) and [website](https://airflowsummit.org/sessions/how-airbnb-twitter-lyft-use-airflow/)} (Airflow Summit 2020)
-- Data DAGs with lineage for fun and for profit by [Bolke de Bruin](https://github.com/bolkedebruin) { [website](https://airflowsummit.org/sessions/data-dags-with-lineage/)} (Airflow Summit 2020)
+- Data DAGs with lineage for fun and for profit by [Bolke de Bruin](https://github.com/bolkedebruin) {[website](https://airflowsummit.org/sessions/data-dags-with-lineage/)} (Airflow Summit 2020)
```
Comment (ttannis) [inline README.md:146]: @feng-tao is this talk related to/mentioning Amundsen?

Comment (feng-tao) [reply README.md:146]: yes, Bolke gave a long intro about Amundsen and Atlas :)
